ctl/src/command.rs

```diff
@@ -1275,7 +1270,7 @@ pub fn query_application(mut channel: Channel<CommandRequest,CommandResponse>, j
     }
 
     let query_domain = QueryApplicationDomain {
-      hostname: splitted.get(0).expect("Domain can't be empty").clone(),
+      hostname: splitted.get(0).context("Domain can't be empty")?.clone(),
```

Comment:
if possible, could you use the method `with_context`

Reply:
done!